### DISCUSSION TITLE
Workflow for more complex docker-compose deploys - how to add additional folders etc...

### DISCUSSION THREAD

**Discussion by dreaded369** [coollabsio/coolify#1458] [Q&A]
I'm just wondering how to deal with docker-compose projects that require more than a yaml file and some environmental variables? I have a few that depend on assets that are stored seperately, and a few I might migrate to coolify that have mounts with data already.
It looks like I'd have to deploy the docker-compose.yml, then stop it, log in to my VPS and add the files/folders there. Is that the only way?
What happens to those assets if the container is deleted? 

Thanks.

**Comment by AKRking** [comment]
same here, have u figured it out?
